Makefile

```diff
@@ -1,7 +1,7 @@
 all: install_requirements configure_host launch_mgmt_cluster
 
 install_requirements:
-	./01_install_requirements.sh
+	./ubuntu_requirement_installation.sh
```

Comment:
This will break it for CentOS, since you call apt for example in the ubuntu_* script

Reply:
Yes, I'm working on this case. I plan to put some bash code here to identify which OS is running

Reply:
I have made the dev-env running on both Ubuntu and CentOS at the moment. Now I will try to run the dev-env on a fresh new CentOS VM

Reply:
Please check the latest commits.